src/power.jl

```diff
@@ -65,6 +65,15 @@ mutable struct IntervalMatrixPower{T}
         @assert k >= 1 "matrix powers must be positive"
         return new{T}(M, Mᵏ, k)
     end
+
+    function IntervalMatrixPower(M::IntervalMatrix{T}, k::Int) where {T}
+        @assert k >= 1 "matrix powers must be positive"
+        pow = IntervalMatrixPower(M, M, 1)
+        @inbounds for i in 1:k
```

Comment:
The right-most index should go to k-1. It is precisely for this reason that we should write a function power(M, k) = get(...)? (I did the same mistake..)

I mean, in addition go this constructor, a function to raise M to the power of k would be useful, returning the interval matrix.

Reply:
I do not understand. I initialize it with 1 and then `increment!` will increment it.